### PR TITLE
BAH Spool Template Education for form 0994

### PR DESCRIPTION
**Pull request by abhijitvk**
## Description of change

PR includes the following updates for new edu form 22-0994.  Form PDF [22-0994](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Disability%20Compensation/BAH-526/engineering/VetTec/Form22-0994.pdf)
- Model changes
- New spool template and create spool file
- Reporting updates for daily fiscal year
- Unit tests

## Testing done
- Unit testing
- Local testing

## Testing planned
- Unit testing
- Cross-browser (by QA)
- Manual testing (by QA)

## Acceptance Criteria (Definition of Done)
- Unit tests passed
    - spec/models/education_benefits_submission_spec.rb
    - spec/models/education_benefits_claim_spec.rb
    - spec/jobs/education_form/forms/va0994_spec.rb
    - spec/jobs/education_form/create_daily_year_to_date_report_spec.rb
    - spec/jobs/education_form/create_daily_spool_files_spec.rb
    - spec/jobs/education_form/create_daily_fiscal_year_to_date_report_spec.rb

#### Unique to this PR
 NA

#### Applies to all PRs

- [X] Appropriate logging
- [X] Swagger docs have been updated, if applicable
- [X] Provide link to originating GitHub issue, or connected to it via ZenHub
- [X] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [X] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

**Comment by abhijitvk** [comment]
Code changes completed for the new edu form 22-0994. @randyhoffmastergit to confirm if the work can move forward. Next step is to get it approved internally and then add AdHoc reviewers for approval.

**Comment by abhijitvk** [comment]
@department-of-veterans-affairs/code-reviewers-for-external-teams-group - Requesting a code review on this PR. Thanks

File: app/workers/education_form/education_facility.rb [hunk 1]
```diff
@@ -68,8 +68,8 @@ def self.facility_for(region:)
     def self.region_for(model)
       record = model.open_struct_form
       address = routing_address(record, form_type: model.form_type)
-      # special case Philippines
-      return :western if address&.country == 'PHL' || model.form_type == '0993'
+
+      check_special_cases(address, model)
```
Comment (lihanli) [inline app/workers/education_form/education_facility.rb]: this isn't going to return here

Comment (abhijitvk) [reply app/workers/education_form/education_facility.rb]: Good catch. Thanks. Its been updated now

**Pull request merged**